content/guides/migrating.md

```diff
@@ -266,13 +266,15 @@ It's no longer necessary to specify it in configuration.
 ```diff
 module: {
   rules: [
-    test: /.css$/,
--    loader: ExtractTextPlugin.extract("style-loader", "css-loader", { publicPath: "/dist" })
-+    loader: ExtractTextPlugin.extract({
-+      fallback: "style-loader",
-+      use: "css-loader",
-+      publicPath: "/dist"
-+    })
+    {
+      test: /.css$/,
+-      loader: ExtractTextPlugin.extract("style-loader", "css-loader", { publicPath: "/dist" })
++      loader: ExtractTextPlugin.extract({
```

Comment:
is there any syntactic difference except `loader` -> `use`?

I'm new to Webpack 2 so I don't wanna assume.